.gitignore

```diff
@@ -8,10 +8,10 @@ src/**/*.css
 .idea
 # IDE - VSCode
 .vscode/*
-!.vscode/settings.json
-!.vscode/tasks.json
-!.vscode/launch.json
-!.vscode/extensions.json
+.vscode/settings.json
```

Comment:
Not sure we should be ignoring these since they are setup for this project.  If you made project changes, would recommend unchecking them out and setup as user settings.